UPDATING.md

```diff
@@ -36,6 +36,8 @@ assists people when migrating to a new version.
   or should have tried to support docker-compose for production use cases, we now actively
   have taken a stance against supporting it. See the PR for details.
 
+- [27777](https://github.com/apache/superset/pull/27777): Changes the default `LOG_LEVEL` from `DEBUG` to `INFO`.
```

Comment:
this notice is already done here https://github.com/apache/superset/pull/28134/files#diff-a35d90f22c0fe6afe6e6999c4291dc377ec9177a3aae2d1c194fe47f34433fadR44